src/convertToObject.js

```diff
@@ -6,7 +6,45 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+
+  const cleanString = sourceString.trim();
+
+  const rules = cleanString.split(';');
+
+  rules.forEach((rule) => {
+    const [rawKey, rawValue] = rule.split(':');
+
+    if (rawKey && rawValue) {
+      const key = rawKey.trim();
+      let value = rawValue.trim();
+
+      if (value.includes(',')) {
+        const lines = value.split(',').map((line) => line.trim());
+
+        value = lines
+          .map((line, index) => {
+            if (index === 0) {
+              return line;
+            } else {
+              return `          ${line}`;
+            }
+          })
+          .join(',\n');
+      }
```

Comment:
Why split values by "," just to join them by "," in the end? If you whish to remove the space from the beginning and the end of the string simply using `.trim` in value should suffice...

This entire `if` block is unnecessary, can you see why?

Reply:
I guess I ended up getting confused and, in trying to pass the tests, I made simple things complicated.